web-app/src/pages/tournament/TournamentsList.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <grid-list-mobile
+      :data="data"
+      :columns="columns"
+      :filter="filter"
+      :expanded="expanded"
+      :pagination="pagination"
+    />
+    <grid-list-desktop
+      :data="data"
+      :columns="columns"
+      :filter="filter"
+      :expanded="expanded"
+      :pagination="pagination"
+    />
```

Comment:
1. Move here screen size condition `v-if ... v-else`, it would be more readable
2. `GridList...` its strange name. `Grid` is not important. I suggest renaming this to `TournamentList...`

Reply:
1. It's 100% readable because it is CALLED `grid-list-mobile` and `grid-list-desktop`. 
2. It is coming from tournament folder so it is obvious it is for tournament. GridList makes it more clear what type of list It actually is. No need to overwrite word "Tournament"

Reply:
1. When i open this i didn't know how it's work, when You want to 'run' mobile and desktop in the same time, so here's my suggestion. if condition will tell us more
2. I think grid or not, type of table is not important, anyway ok